core/chains/evm/logpoller/parser.go

```diff
@@ -432,6 +432,12 @@ func orderToString(dir query.SortDirection) (string, error) {
 	}
 }
 
+// MakeContractReaderCursor is exported to ensure cursor structure remains consistent.
+func MakeContractReaderCursor(log Log) string {
```

Comment:
nit/ `Format` is a little more typical for returning a `string`
```suggestion
func FormatContractReaderCursor(log Log) string {
```